kinfer/protos/kinfer.proto

```diff
@@ -154,18 +173,34 @@ message IMUValue {
     IMUMagnetometerValue magnetic_field = 3;
 }
 
+message AngularVelocitySchema {
+}
+
+message AngularVelocityValue {
+    float x = 1;
+    float y = 2;
+    float z = 3;
+}
+
+message EulerRotationSchema {
+}
+
+message EulerRotationValue {
+    float x = 1;
+    float y = 2;
+    float z = 3;
+}
```

Comment:
i believe that using the euler rotation and angular velocity is a bad idea. in sim we should be training with imu values directly rather than with these values - i think this will transfer better sim2real